### ISSUE TITLE
Time Accounting: Repeatedly pressing Enter duplicates the article

### ISSUE THREAD

**Issue by cautopartes** [zammad/zammad#5326]
### Used Zammad Version

6.3.1 and 6.4.0

### Environment

- Installation method: any
- Operating system: any
- Database + version: any
- Elasticsearch version: any
- Browser + version: any

### Actual behaviour

* With active time accounting
* When updating a ticket with an article and pressing "Enter" repeatedly (quickly)
* The article and a duplicate of it are inserted

### Expected behaviour

* New articles should be added only once to the ticket 

### Steps to reproduce the behaviour

* Activate time accounting
* Update a ticket with an article as an agent (any channel)
* Insert value for accounted time
* Press "Enter" repeatedly in quick succession
* The ticket is updated with two versions of the same article

### I'm sure this is a bug and no feature request or a general question.

yes

**Comment by MrGeneration** [comment]
I was not able to verify this yesterday in Vivaldi and Firefox, what did you do differently?

**Comment by dvuckovic** [comment]
I managed to reproduce it, by quickly pressing Enter twice in the input field (with _0_ as the value, you can use whichever). It could be it's present only on systems with a slight delay after the submission, as we are not disabling this particular form.

Ref: Ticket#10161002
